generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -80,7 +80,7 @@ gradle-test:
 
 gradle-integration-test:
 <%_ if (insideDocker) { _%>
-    <%_ if (cacheProvider === 'redis' || ['cassandra', 'couchbase', 'neo4j'].includes(prodDatabaseType)) { _%>
+    <%_ if (cacheProvider === 'redis' || databaseType === "sql" || ['cassandra', 'couchbase', 'neo4j'].includes(prodDatabaseType)) { _%>
```

Comment:
Very minor thing, but we can simplify it to 

`<%_ if (cacheProvider === 'redis' || ['sql', 'cassandra', 'couchbase', 'neo4j'].includes(databaseType)) { _%>`

I think